src/App.jsx

```diff
@@ -3,26 +3,62 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodosApi } from './api/api';
 
 class App extends React.Component {
   state = {
-    todos: [],
+    todos: null,
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodosApi().then(response => (
+      this.setState({
+        todos: response.data,
+      })
+    ));
+  }
+
+  getSelectedUserId = (userId) => {
+    if (this.state.selectedUserId === userId) {
```

Comment:
```suggestion
  getSelectedUserId = (userId) => {
    const { selectedUserId } = this.state;
  
    if (selectedUserId === userId) {
```
Деструктуризируй значения из state